client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/WebClientFactory.java

```diff
@@ -160,9 +160,6 @@ private WebClient createSecureWebClient(String baseUrl, String token, String tru
     }
     ClientHttpConnector connector = new ReactorClientHttpConnector(client);
     WebClient.Builder webClientBuilder = createWebClientBuilder();
-    if (token != null) {
-      webClientBuilder = webClientBuilder.defaultHeaders(h -> h.setBearerAuth(token));
-    }
```

Comment:
What code path exercises this?

Reply:
WebClient is used in ApiClientFactory Only to create ApiClients. 
Reason to remove is ApiClient also adds the header to webClient headers while preparing the request, so I wanted to avoid case when webClient header is not overridden by ApiClient's default in anyway.

Reply:
@rohitkum2506 You can actually do some negative test like not passing the token and observe the errors. You can also also perform some table crud operations etc.

Reply:
Another way to test is to verify working of jobs/job scheduler.

Reply:
Jobs scheduler also uses the same token file to initiate the ApiClient. An independent jobs scheduler tests makes sense since we are changing the ApiClient for Jobs client as well but I think it provides same validations as local Docker tests.